package.json

```diff
@@ -59,7 +58,6 @@
     "ember-load-initializers": "^1.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-qunit": "^3.4.1",
-    "ember-resolver": "^5.0.1",
```

Comment:
just curious why ember-resolver got dropped?

Reply:
because i'm an idiot.

i _thought_ it was added in https://github.com/Esri/torii-provider-arcgis/pull/36/, but i can see now that it was just shuffled a bit.